docs/spec/spec-proposals/f1-fee-distribution/f1_fee_distr.tex

```diff
@@ -68,12 +88,12 @@ \subsection{Commission Rates}
 
 In block $h$ a validator receives $f_h$ fees.
 Instead of incrementing that validators ``total accrued fees this period variable" by $f_h$, it is instead incremented by $(1 - commission\_rate) * f_p$.
-Then $commission\_rate * f_p$ is deposited directly to the validator.
-This scheme allow for updates to a validator's commission rate every block if desired.
+Then $commission\_rate * f_p$ is deposited directly to the validator's account.
+This allows for efficient updates to a validator's commission rate every block if desired.
 
 \subsection{Slashing}
 \label{ssec:slashing}
-Slashing is distinct from withdrawals, since not only does it lower the validators total amount of stake, but it also lowers each of its delegator's stake by a fixed percentage.
+Slashing is distinct from withdrawals, since lowers the stake of all of the delegator's by a fixed percentage.
 Since noone is charged gas for slashes, a slash cannot iterate over all delegators.
```

Comment:
```suggestion
Since no one is charged gas for slashes, a slash cannot iterate over all delegators.
```

Reply:
also, I don't know if I agree, slashes do have a high cost, they're unlikely to be spammable

Reply:
On the later stage cosmos hub, not in all zone configurations. Imagine a zone with shared security for 99 validators, and then one other random person becomes a validator. They can just keep on getting slashed.